src/quo2/components/avatars/wallet_user_avatar/view.cljs

```diff
@@ -0,0 +1,56 @@
+(ns quo2.components.avatars.wallet-user-avatar.view
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.avatars.wallet-user-avatar.style :as style]
+            utils.string))
+
+(def circle-sizes
+  {:small   20
+   :medium  32
+   :large   48
+   :x-large 80})
+
+(def font-sizes
+  {:small   :label
+   :medium  :paragraph-2
+   :large   :paragraph-1
+   :x-large :heading-1})
+
+(def font-weights
+  {:small   :medium
+   :medium  :semi-bold
+   :large   :semi-bold
+   :x-large :medium})
+
+(defn- view-internal
+  "Options:
+
+  :full-name - string (default: nil) - used to generate initials
+  :color  - keyword (default: nil) - color of the avatar
+  :size   - keyword (default: :x-large) - size of the avatar"
+  [{:keys [full-name color size theme]
```

Comment:
For 'color' we refer to this as 'customisation-color'. Can you adjust for consistency? 
Additionally if the colors doc didn't include that perhaps you could update it to mention that 😃